bevy_console_parser/src/value.rs

```diff
@@ -44,17 +44,6 @@ impl From<ValueRaw<'_>> for Value {
     }
 }
 
-// impl ValueRawOwned {
-//     pub fn as_value_raw(&self) -> ValueRaw<'_> {
-//         match self {
-//             ValueRawOwned::String(s) => ValueRaw::String(s),
-//             ValueRawOwned::Int(num, raw) => ValueRaw::Int(num, &raw),
-//             ValueRawOwned::Float(num, raw) => ValueRaw::Float(num, &raw),
-//             ValueRawOwned::Bool(b, raw) => ValueRaw::Bool(b, &raw),
-//         }
-//     }
-// }
-
```

Comment:
I think this was the commented code I saw :)

Reply:
Ah okay, I've removed that in a recent commit